src/test/java/com/flightstats/hub/util/HubUtilsTest.java

```diff
@@ -8,11 +8,11 @@
 @Execution(ExecutionMode.SAME_THREAD)
 class HubUtilsTest {
 
-    private static final String HUT_TEST = "test_0_HubUtilsTest" + StringUtils.randomAlphaNumeric(6);
+   /* private static final String HUT_TEST = "test_0_HubUtilsTest" + StringUtils.randomAlphaNumeric(6);
     private static HubUtils hubUtils;
     private static String channelUrl;
 
-    /*@BeforeAll
+    @BeforeAll
     static void setUpClass() {
         hubUtils = IntegrationTestSetup.run().getInstance(HubUtils.class);
         channelUrl = create();
```

Comment:
Should this be commented out?  Seems like we should fix the tests, delete them, or at-ignore them if we're going to fix them soon so we at least get compile-time checking.

Reply:
I removed the comment. This test sometimes takes very long time to finish execution (i think it is due to timeout settings). So I generally comment it out. There is a ticket open to look into this issue.

Reply:
Ah, yeah, I have that problem on my box.  Running the tests serially instead of in parallel is the work-around I've been using.